### PR TITLE
Add option to hide stdout when managing repos

### PR DESCRIPTION
**Pull request by elyezer**
Create the `_silencer` decorator which helps on receiving the `silent`
keyword argument. If `silent` is `True` the task will run within the
hide context manager.

### CODE REVIEW

File: automation_tools/repository.py [hunk 1]
```diff
@@ -135,15 +161,19 @@ def enable_satellite_repos(cdn=False, beta=False, disable_enabled=True):
...
     If yum-config-manager is available this task will disable the repos, if not
     it will move the beaker repo files to the running user home directory
 
+    If the keyword argument ``silent`` is ``True`` then the stdout output will
+    be hidden.
+
     """
     # Clean up system if Beaker-based
-    result = run('which yum-config-manager', warn_only=True)
+    result = run('which yum-config-manager', quite=True)
```
Comment (omaciel) [inline automation_tools/repository.py]: s/quite/quiet?

Comment (elyezer) [inline automation_tools/repository.py]: :+1:

**Comment by omaciel** [comment]
ACK

File: automation_tools/repository.py [hunk 2]
```diff
@@ -135,15 +161,19 @@ def enable_satellite_repos(cdn=False, beta=False, disable_enabled=True):
...
     If yum-config-manager is available this task will disable the repos, if not
     it will move the beaker repo files to the running user home directory
 
+    If the keyword argument ``silent`` is ``True`` then the stdout output will
+    be hidden.
+
     """
     # Clean up system if Beaker-based
-    result = run('which yum-config-manager', warn_only=True)
+    result = run('which yum-config-manager', quiet=True)
```
Comment (Ichimonji10) [inline automation_tools/repository.py]: This is supposed to be "quiet" and not "silent", right?

**Comment by Ichimonji10** [comment]
ACK

**Pull request merged**